js/src/components/campaign-conversion-notice/report-notice.test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * Internal dependencies
+ */
+import CampaignConversionReportsNotice from '.~/components/campaign-conversion-notice/reports-notice';
+import getConversionCampaignStatusNotice from '.~/utils/getConversionCampaignStatusNotice';
+
+jest.mock( '.~/utils/getConversionCampaignStatusNotice', () => jest.fn() );
+
+describe( 'Notice Reports Campaign Migration', () => {
```

Comment:
💅  This test file's name is `report-notice.test.js` while the filename of the component it's testing on is `reports-notice.js`.